### ISSUE TITLE
Code refactor

### ISSUE THREAD

**Issue by reteps** [AtomicCoding/Quizlet-Bot#13]
everyone's favorite thing! I believe you really need to refactor your code. A 3000 line file should be broken up, and in multiple spots you have redundant code. I'll point them out.

**Comment by yeSpud** [comment]
I mean, that & if it's done by bot, it's easier to update them ~~if~~ when quizlet changes how the activities work

**Comment by reteps** [comment]
i'm gonna make a bot similar to this one, stay tuned.

**Comment by YonatanAhituv** [comment]
@reteps Yeah, I was thinking of doing something like this for a while now. I'd like to keep this on the same repo. Would you be interested in joining as a contributor and working on a `dev` branch together?

**Comment by reteps** [comment]
sorry, i made a new repo. quizizz and kahoot branches are complete, working on quizlet.
